src/core/lib/address_utils/sockaddr_utils.cc

```diff
@@ -224,6 +247,14 @@ absl::StatusOr<std::string> grpc_sockaddr_to_string(
   }
 #endif
 
+#ifdef GRPC_HAVE_VSOCK
+  if (addr->sa_family == GRPC_AF_VSOCK) {
+    const sockaddr_vm* addr_vm = reinterpret_cast<const sockaddr_vm*>(addr);
+    out = std::string(absl::StrCat(addr_vm->svm_cid, ":", addr_vm->svm_port));
```

Comment:
`absl::StrCat()` returns `std::string`, so there's no need to create a new `std::string` from the result.